src/app/public/modules/file-attachment/fixtures/file-drop-control.compoennt.fixture.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Component,
+  OnInit,
+  ViewChild
+} from '@angular/core';
+
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
+import {
+  SkyFileDropControlComponent
+} from '../file-drop-control.component';
+
+@Component({
+  selector: 'file-drop-control-test',
+  templateUrl: './file-drop-control.component.fixture.html'
+})
+export class FileDropControlTestComponent implements OnInit {
+
+  public attachment: FormControl;
```

Comment:
It looks like `required` and `showLabel` aren't being used?

Reply:
Removed